pages/28.Switching to Composer/default.en.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Switch to Composer'
+slug: switch-to-composer
+twitterenable: true
+twittercardoptions: summary
+articleenabled: false
+orgaenabled: false
+orga:
+    ratingValue: 2.5
+orgaratingenabled: false
+personenabled: false
+facebookenable: true
+---
+
+---
+
+Up until Mautic 4, users could simply download Mautic as a ZIP file and install it on any PHP server. However, many users were running into installation and update errors (TODO maybe link background article?). Next to that, Mautic recently introduced the Marketplace (TODO link to docs page on Marketplace) which isn't compatible with this installation method.
+
+For the reasons mentioned above, Mautic will require all users to use Composer-based installations starting from Mautic 5.
+
+### How do I switch to a Composer-based installation?
+
+Before we start, it's good to understand that there's two aspects to Mautic:
+- The database. Here, all your contact data etc. is stored.
+- The codebase. This is where Mautic interacts with the database.
+
+When switching to a Composer-based installation, the **database** will not be touched. You will only be replacing the **codebase**.
+
+In this tutorial, we'll assume that Mautic is currently installed in `/var/www/html`.
+
+Here's the steps you will need to follow to switch to a Composer-based installation:
+
+1. Go to `/var/www`
+1. Run `composer create-project mautic/recommended-project:4.x-dev html-new --no-interaction`
+1. Copy the following files and folders from `/var/www/html` to `/var/www/html-new`:
+    - Configuration files - in most cases this can be found at `app/config/local.php`
```

Comment:
```suggestion
    - Configuration files - in most cases this can be found at `app/config/local.php` and should be moved to `docroot/app/config/local.php`
```